src/containers/products/styled/StyledProductComponents.js

```diff
@@ -233,83 +201,84 @@ const ProductOverviewDescriptionWrapper = styled.span`
   }
 `;
 
-const FeatureShot = styled.span`
-  align-self: center;
-  background-image: url(${(props) => props.bgImage});
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: contain;
-  border-radius: 2px;
-  border: 1px solid ${NEUTRALS.GRAY_09};
-  box-shadow: 0px 10px 30px rgba(48, 47, 57, 0.05);
-  box-sizing: border-box;
-  height: 556px;
-  width: 480px;
+const ExtendedFeatureHeader = styled(SectionHeader)`
+  align-self: flex-start;
+  text-align: left;
+  width: 100%;
 
-  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
-    height: 348px;
-    width: 300px;
+  @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
+    width: 432px;
   }
 `;
 
-const PermissionsFeatureShot = styled(FeatureShot)`
-  height: 315px;
-
-  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
-    height: 197px;
-  }
+const ExtendedFeatureDescription = styled.div`
+  font-size: 18px;
+  position: relative;
+  top: auto;
+  ${paragraphStyles}
 
-  @media only screen and (min-width: ${MEDIA_QUERY_MD}px) {
-    height: 315px;
+  @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
+    top: auto;
+    width: 464px;
   }
 `;
 
-const ProductOverviewTitle = styled(FeatureOverviewTitle)`
-  font-size: 30px;
-  line-height: 140%;
+const ProductFeaturesContent = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
   width: 100%;
 
-  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
-    font-size: 20px;
+  @media only screen and (max-width: ${MEDIA_QUERY_MD}px) {
+    margin-top: 300px;
   }
 
   @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
-    font-size: 36px;
+    align-items: flex-start;
+    flex-direction: row;
+    margin: 440px auto 96px auto;
+    width: ${CONTENT_WIDTH}px;
+  }
+
+  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
+    margin-top: 150px;
   }
 `;
 
-const ProductOverviewDescription = styled(FeatureOverviewDescription)`
-  font-size: 18px;
-  margin-top: 24px;
-  position: relative;
-  top: auto;
+const ExtendedFeaturesContent = styled(ProductFeaturesContent)`
+  align-self: center;
+  margin-top: 0;
+
+  @media only screen and (max-width: ${MEDIA_QUERY_MD}px) {
+    margin-top: 32px;
+  }
 
   @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
-    top: auto;
+    align-items: flex-start;
+    flex-direction: column;
+    margin: 0 0 200px 0;
```

Comment:
`margin: 0 200px 0`

Reply:
actually i think it should be `margin: 0 0 200px;`. but good call!